src/ServicePulse.Host/app/js/views/redirect/service.js

```diff
@@ -0,0 +1,105 @@
+﻿; (function (window, angular, undefined) {
+    'use strict';
+    
+    function service($http, $timeout, $q, $rootScope, $interval, scConfig, uri, notifications, notifyService) {
+        var notifier = notifyService();
+
+        var redirects = {
+            total :0,
+            data: []
+        };
+
+        function getData() {
+            var url = uri.join(scConfig.service_control_url, 'redirects');
+            return $http.get(url).then(function (response) {
+                notifier.notify('RedirectsUpdated', { total: response.headers('Total-Count'), data: response.data });
+                notifier.notify('RedirectMessageCountUpdated', response.headers('Total-Count'));
+            });
+        }
+
+        var redirectsUpdatedTimer = $interval(function () {
+            getData();
```

Comment:
Is this really something we have to query? Why can't we use SignalR to tell us when it changes?